PlayCover/Views/Uninstaller.swift

```diff
@@ -149,5 +154,18 @@ class Uninstaller {
                 Log.shared.error(error)
             }
         }
+
+        for url in otherPruneURLs {
+            do {
```

Comment:
Also I think the do/catch statement can be put outside the for loop and be combined with the do/catch statement above (the `pruneURLs` for loop).

Reply:
Done.